### PR TITLE
Android bundle

### PR DESCRIPTION
**Pull request by pazos**
- adaptative icons
- minify + shrink
- remove unused assets to save a few KBs
- install apk by clicking on notification after download (doesn't apply to F-Droid)

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6870)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -358,14 +358,31 @@ androidupdate: all
 	# assets are compressed manually and stored inside the APK.
 	cd $(INSTALL_DIR)/koreader && 7z a -l -mx=9 -mfb=256 -mmt=on \
 		../../$(ANDROID_LAUNCHER_DIR)/assets/module/koreader-$(VERSION).zip * \
+		-xr!*cache$ \
+		-xr!*clipboard$ \
+		-xr!*data/dict$ \
+		-xr!*data/tessdata$ \
+		-xr!*history$ \
+		-xr!*l10n/.tx$ \
```
Comment (Frenzie) [inline Makefile:366]: We can just remove that file. Switching between Weblate and Transifex isn't a simple binary thing anyway, see https://github.com/koreader/koreader/issues/6775.

File: Makefile [hunk 2]
```diff
@@ -358,14 +358,31 @@ androidupdate: all
...
 		-xr!*resources/fonts* \
 		-xr!*resources/icons/src* \
+		-xr!*rocks/bin$ \
+		-xr!*rocks/lib/luarocks$ \
+		-xr!*screenshots$ \
 		-xr!*share/man* \
 		-xr!*spec$ \
+		-xr!*tools$ \
 		-xr!*COPYING$ \
+		-xr!*LICENSE$ \
```
Comment (Frenzie) [inline Makefile:378]: I probably overlooked that before, but is that a good idea to skip?

Comment (pazos) [reply Makefile:378]: There are a bunch of LICENSE files, all buried in /data/data/org.koreader/launcher/files, which are not available to end users.

Probably a generic license, like https://github.com/koreader/koreader/blob/master/platform/debian/copyright, on top of the APK is more useful than something buried into assets and extracted to private app dirs.

Comment (pazos) [reply Makefile]: All LICENSE files re-included for now. It is a major PITA to aknowledge all licenses in single document.

File: Makefile [hunk 3]
```diff
@@ -358,14 +358,29 @@ androidupdate: all
...
 		-xr!*COPYING$ \
+		-xr!*Makefile$ \
+		-xr!*NOTES.txt$ \
+		-xr!*NOTICE$ \
 		-xr!*README.md$ \
-		-xr!git$ \
-		-xr!gitiginore$ 
+		-xr!*koreader.icns$ \
+		-xr!*.git$ \
+		-xr!*.gitignore$
```
Comment (Frenzie) [inline Makefile:383]: Or maybe something along the lines of `-xr'!.*'`, unless there's something hidden we do want?

**Review by Frenzie — APPROVED** [review]
lgtm

File: Makefile (hunk 3 above)
Comment (pazos) [reply Makefile]: yeah, makes sense

**Pull request merged**